app/controllers/services/service_instances_controller.rb

```diff
@@ -119,6 +119,7 @@ def update(guid)
         raise CloudController::Errors::ApiError.new_from_details('UserProvidedServiceInstanceHandlerNeeded')
       end
 
+      validate_not_shared_space_update(service_instance)
```

Comment:
I have a bit of a nit pick around naming. In general, we prefer to use positive assertions rather than negatives (not, un, etc.) because it's easier for us to reason about. I propose we change this method name to something like `validate_shared_space_updateable` or `validate_shared_space_writeable`.

Reply:
Yep, sounds much better.